conf/shares.routes

```diff
@@ -11,7 +11,12 @@ POST        /:srn/change-total-value-quoted-shares
 GET         /:srn/what-you-will-need-shares                                             controllers.nonsipp.shares.WhatYouWillNeedSharesController.onPageLoad(srn: Srn)
 GET         /:srn/submit-what-you-will-need-shares                                      controllers.nonsipp.shares.WhatYouWillNeedSharesController.onSubmit(srn: Srn)
 
-GET         /:srn/which-shares-held-transaction/:index                                          controllers.nonsipp.shares.TypeOfSharesHeldController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = NormalMode)
-POST        /:srn/which-shares-held-transaction/:index                                          controllers.nonsipp.shares.TypeOfSharesHeldController.onSubmit(srn: Srn, index: Max5000, mode: Mode = NormalMode)
-GET         /:srn/change-which-shares-held-transaction/:index                                   controllers.nonsipp.shares.TypeOfSharesHeldController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = CheckMode)
-POST        /:srn/change-which-shares-held-transaction/:index                                   controllers.nonsipp.shares.TypeOfSharesHeldController.onSubmit(srn: Srn, index: Max5000, mode: Mode = CheckMode)
\ No newline at end of file
+GET         /:srn/which-shares-held-transaction/:index                                  controllers.nonsipp.shares.TypeOfSharesHeldController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = NormalMode)
+POST        /:srn/which-shares-held-transaction/:index                                  controllers.nonsipp.shares.TypeOfSharesHeldController.onSubmit(srn: Srn, index: Max5000, mode: Mode = NormalMode)
+GET         /:srn/change-which-shares-held-transaction/:index                           controllers.nonsipp.shares.TypeOfSharesHeldController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = CheckMode)
+POST        /:srn/change-which-shares-held-transaction/:index                           controllers.nonsipp.shares.TypeOfSharesHeldController.onSubmit(srn: Srn, index: Max5000, mode: Mode = CheckMode)
+
+GET         /:srn/why-does-scheme-hold-shares/:index                                  controllers.nonsipp.shares.WhyDoesSchemeHoldSharesController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = NormalMode)
+POST        /:srn/why-does-scheme-hold-shares/:index                                  controllers.nonsipp.shares.WhyDoesSchemeHoldSharesController.onSubmit(srn: Srn, index: Max5000, mode: Mode = NormalMode)
+GET         /:srn/change-why-does-scheme-hold-sharesn/:index                           controllers.nonsipp.shares.WhyDoesSchemeHoldSharesController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = CheckMode)
```

Comment:
Looks like a rogue character (n) snuck into the url here

Reply:
Good spot @ccatosdevelopment , we are not using change- URLs per now, so I'll change it with the next ticket.